avaframe/in3Utils/cfgHandling.py

```diff
@@ -497,3 +504,53 @@ def errorDuplicateListEntry(listKeys, message):
     if len(listKeys) != len(list(set(listKeys))):
         log.error(message)
         raise AssertionError
+
+
+def rewriteLocalCfgs(cfgFull):
+    """ fetch all override sections in cfgFull and write a local_NAMEOVERRIDE.ini configuration file for the
+        available sections - caution this overrides existing local configuration files
+        naming is collection_module_override
+        where package refers to e.g. ana1Tests, ana3AIMEC, etc-
+        and module to e.g. energyLineTest.py, ana3AIMEC.py so all python files inside the packages that have a nameCfg.ini file too
+
+         Parameters
+         -----------
+         cfgFull: configparser
+            configuration with override sections for modules
+
+    """
+
+    for section in cfgFull.sections():
+        if '_override' in section:
```

Comment:
generate a list first of these section and then loop over that one -> to reduce the complexity of this code
This is hard to read...